src/main/scala/wdl4s/expression/TypeEvaluator.scala

```diff
@@ -65,10 +65,9 @@ case class TypeEvaluator(override val lookup: String => WdlType, override val fu
           val message = failures.collect { case f: Failure[_] => f.exception.getMessage }.mkString("\n")
           Failure(new WdlExpressionException(s"Could not evaluate expression:\n$message"))
         case (successes, _) =>
-          for {
-            keyType <- WdlType.homogeneousTypeFromTypes(evaluatedMap map { case (k, v) => k.get})
-            valueType <- WdlType.homogeneousTypeFromTypes(evaluatedMap map { case (k, v) => v.get})
-          } yield WdlMapType(keyType, valueType)
+          val keyType = WdlType.homogeneousTypeFromTypes(evaluatedMap map { case (k, v) => k.get})
```

Comment:
i know you inherited it but can you add the space prior to the `}` on these? the space on one side but not the other is jarring :(